### DISCUSSION TITLE
希望增加宽屏模式下的选集功能

### DISCUSSION THREAD

**Discussion by fate1ess** [the1812/Bilibili-Evolved#2920] [功能建议 / Ideas]
宽屏模式没有选集，希望加一个功能，谢谢

**Comment by Geeyun-JY3** [comment]
宽屏不是往下滚一点就能看到了吗……意思是做进播放器控件里？

**Reply by fate1ess** [reply]
是的，就是这个意思

**Reply by XMB-7** [reply]
能否考虑新增两个快捷键, 切换到上一p和下一p, 目前的快捷键组件里面没有支持.
这样宽屏播放一p完毕, 通过快捷键即可播放下一p, 免去向下滚动寻找点击下一p了.

**Reply by Geeyun-JY3** [reply]
> 能否考虑新增两个快捷键, 切换到上一p和下一p, 目前的快捷键组件里面没有支持. 这样宽屏播放一p完毕, 通过快捷键即可播放下一p, 免去向下滚动寻找点击下一p了.

如果只是需要这两个功能的快捷键，b 站目前就已经有这样的快捷键：

- <kbd>]</kbd>：下一个
- <kbd>[</kbd>：上一个

这俩目前确实是还没做进 `快捷键扩展` 这功能里，没法自定义：https://github.com/the1812/Bilibili-Evolved/discussions/1646#discussioncomment-479874
